index.html

```diff
@@ -1,110 +1,164 @@
+
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0"
+    />
+    <meta name="HandheldFriendly" content="true" />
+    <meta name="mobile-web-app-capable" content="yes" />
+
     <title>FRONTEND CHALLENGE</title>
-    <link href="style.css" rel="stylesheet" type="text/css" />
-    <link href="https://www.fontify.me/wf/7116669ff62ea6ce905b50766eaca6b8" rel="stylesheet" type="text/css">
-    <link href="https://www.fontify.me/wf/e6a3c17c14d6295d075e29aa38e455ce" rel="stylesheet" type="text/css">
-    <link href="https://www.onlinewebfonts.com/fonts/SF_Compact_Display" rel="stylesheet" type="text/css">
-      </head>
+
+    <link href="./assets/styles/reset.css" rel="stylesheet" type="text/css" />
+    <link href="./assets/styles/style.css" rel="stylesheet" type="text/css" />
+
+    <link
+      href="https://www.fontify.me/wf/7116669ff62ea6ce905b50766eaca6b8"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link
+      href="https://www.fontify.me/wf/e6a3c17c14d6295d075e29aa38e455ce"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link
+      href="https://www.onlinewebfonts.com/fonts/SF_Compact_Display"
+      rel="stylesheet"
+      type="text/css"
+    />
+  </head>
   <body>
-    <section class="container">
-      <nav>
-        <div class="base"></div>
-        <div class="backarrow"> </div>
-            <h1> Bus Routes </h1>
+    <!-- Header -->
+    <header>
+      <div class="StatusBar"></div>
+      <nav class="NavigationBar">
+        <img src="./assets/img/back_arrow.png" />
+        <h1>Bus Routes</h1>
+      </nav>
 
-        <div class="input-container">
-          <i class="Search-icon"></i>
-          <img class="Search" src="Search.png">
-          <input class="input-field" type="text" placeholder="State/Washington" name="StWa"
+      <section class="BusSearch">
+        <div class="BusSearch__input">
+          <input type="text" placeholder="State/Washington" name="origin" />
         </div>
-
-        <div class="input-container2">
-          <i class="Search-icon"></i>
-          <img class="Search" src="Search.png">
-          <input class="input-field" type="text" placeholder="" name="StWa">
-          <img class="switcharrows" src="switcharrows.png">
+        <div class="BusSearch__input">
+          <input type="text" placeholder="" name="destination" />
         </div>
-      </nav>
-      <section class="secondhalf">
-        <ul class="container2">
-          <div class="shape"></div>
-          <li class="busnumber">&nbsp;60</li>
-          <li class="col1">Eastbound</li>
-          <li class="col1-2">Randolph/Harbor Dr </li>
-          <li class="col2"><strong>6:25p</strong> ETA</li>
-          <li class="col2-1">every 20 mins</li>
-          <img class="forwardarrow" src="forwardarrow.png">
-        </ul>
+      </section>
+    </header>
+
+    <!-- Bus Routes -->
 
-         <ul class="container2">
```

Comment:
I changed the markup for the routes list quite a bit, it's more "semantic" to have a few list items under `ul` and use `div` for the smaller parts of a list item.